js/qz-tray.js

```diff
@@ -615,7 +615,12 @@ var qz = (function() {
                     //disable secure ports if page is not secure
                     if (typeof location === 'undefined' || location.protocol !== 'https:') {
                         if (options == undefined) { options = {}; }
-                        options.usingSecure = false;
+
+                        //respect forcing secure ports if it is defined, otherwise disable
+                        if (options.usingSecure === 'undefined') {
```

Comment:
Ok.  FYI, PRs need to be fixed and merged or rejected so if you don't fix this on your branch I'll have to close this out and do the commit myself. :+1: